### DISCUSSION TITLE
Embed a specific channel in other tools / applications

### DISCUSSION THREAD

**Discussion by jprusch** [mattermost/community#24] [Enhancements]
The Mattermost client is the aggregation tool for all communication in a company/community, but sometimes we need more context or are just working in another tool where a channel based integration makes more sense.

Some platforms offer micro-blogging on a subject, a project, a team, department. Embedding a defined Mattermost channel as a micro blogging activity flow (maybe read only for some users) and the option for others to comment on a message would be great. This could be implemented with a specific channel URL, which displays the content of this channel matching the look of the app embedding the channel.

Here a sketchup of the idea:
<img width="354" alt="mm" src="https://user-images.githubusercontent.com/9337414/122954778-077f7400-d380-11eb-814c-e9517120badf.png">

Any thoughts?

**Comment by BenLloydPearson** [comment]
I love this idea. We could use this at developer events to display a live feed of activities that we're hosting on our chat server.

**Reply by jprusch** [reply]
Yeah, you still have got the whole discussion in one place (MM), but can fan out channels others have not in focus. Also, Mattermost is (sadly) not used as much as I want it by everyone in our company, so this enhancement would give the usage a real push.

**Comment by chetanyakan** [comment]
Also, embedding individual threads.
